techmatters/flex-plugins
# PreviousContactsBanner displays case and contact count with new client profile endpoint

Primary reviewer:

## Description
- Uses endpoint in https://github.com/techmatters/hrm/pull/471 to check and display cases and contacts count for a contact identifier
- Introduces a new flag: `enable_client_profiles` flag
- This has been tested for web, voice call, and whatsapp channels to see that identifiers work. 

### Checklist
- [n/a] Strings are localized

### Related Issues
Fixes # https://tech-matters.atlassian.net/browse/CHI-2254

### Verification steps
1. Ensure [PR](https://github.com/techmatters/hrm/pull/471) is merged and deployed
2. Start a conversation from any channel
3. Save contact and/or case
4. Start another conversation from the same contact info just used
5. Check the PreviousContactBanner
![Screenshot](https://github.com/techmatters/flex-plugins/assets/102122005/3e7c1de9-15de-4a19-9ea7-a87866c3ccb9)
